blocks.qmd

```diff
@@ -11,21 +11,21 @@ understanding of all the files and folders as well as the reasons we
 have them.
 
 ![Project blocks that make up the Seedcase
-project.](images/project-blocks.svg){width="70%"}
+Project.](images/project-blocks.svg){width="70%"}
 
-While the core of the Seedcase project is the Seedcase software
-"Product" itself, it depends on the proper functioning of additional
-"blocks". The Seedcase Product relies on the two foundation layers,
-which are the "Documentation" about the Product and its use as well as
-the "Culture and Collaboration" that enables both Documentation and
-Product to be developed. Supporting all these layers are the support
-structures that tell "How" and "Why" things are done.
+While the core of the Seedcase Project is the "Software" itself, it also
+depends on the proper functioning of additional "blocks". Seedcase
+Software relies on the two foundation layers, which are the
+"Documentation" about the Software and its use as well as the "Culture
+and Collaboration" that enables both Documentation and Software to be
+developed. Supporting all these layers are the support structures that
+tell "How" and "Why" things are done.
```

Comment:
Here and in the diagram, I put "Software" instead of "Product" to avoid implying that there is a single product. That said, I can see a sort of more abstract, collective reading of "Product" in this context that would cover all software deliverables, so happy to change it back.